website/docs/components/side-panel/side-panel-a11y.md

```diff
@@ -19,10 +19,20 @@ Table: Keyboard support
 
 See detailed information about the keyboard support for the clickable elements in the [Keyboard control guide](/core-principles/a11y/a11y-keyboard#keyboard_support_for_button_link_input_etc).
 
+### Default attributes
+
+Table: Default attributes
+
+| Component               | Attribute               | Usage                                       |
+| ----------------------- | ----------------------- | ---------------------------------------------- |
+| `SidePanel`             | `role="dialog"`         | Identifies the element that serves as the dialog container.     |
+| `SidePanel`             | `aria-modal="true"`     | Tells assistive technologies that the content underneath the current dialog is inert (not available for interaction). |
```

Comment:
Just a little thought. Perhaps we can remove the word "inert" and the parentheses in this sentence.

> Tells assistive technologies that the content underneath the current dialog is not available for interaction.

Reply:
done